test/resources/policy_generator/acm-hardening_subscription.yaml

```diff
@@ -0,0 +1,77 @@
+---
+apiVersion: cluster.open-cluster-management.io/v1beta1
+kind: ManagedClusterSetBinding
+metadata:
+  name: default
+spec:
+  clusterSet: default
+---
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: acm-hardening
+spec:
+  componentKinds:
+    - group: apps.open-cluster-management.io
+      kind: Subscription
+  descriptor: {}
+  selector:
+    matchExpressions:
+      - key: app
+        operator: In
+        values:
+          - acm-hardening
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: Channel
+metadata:
+  annotations:
+    apps.open-cluster-management.io/reconcile-rate: high
+  name: acm-hardening
+spec:
+  type: Git
+  pathname: https://github.com/stolostron/policy-collection.git
+---
+apiVersion: apps.open-cluster-management.io/v1
+kind: Subscription
+metadata:
+  annotations:
+    apps.open-cluster-management.io/git-branch: main
+    apps.open-cluster-management.io/git-path: policygenerator/policy-sets/stable/acm-hardening
+    apps.open-cluster-management.io/reconcile-option: merge
+  labels:
+    app: acm-hardening
+  name: acm-hardening-subscription
+spec:
+  allow:
```

Comment:
allow list is no longer needed

Reply:
I took out the allow list and the tests are working fine.  Thanks!